### PR TITLE
Tma 175 event-processing tests

### PR DESCRIPTION
**Pull request by Aidan23Smith**

### CODE REVIEW

**Review by jashelb90 — APPROVED** [review]
looks good

File: tests/event-processing-tests/gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.8-bin.zip
```
Comment (des-hippo) [inline tests/event-processing-tests/gradle/wrapper/gradle-wrapper.properties:3]: Just curious, is there a reason for not using gradle 7+ ?

File: tests/event-processing-tests/src/test/java/uk/gov/di/txma/eventprocessor/bdd/LambdaToS3.java [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+import java.util.Objects;
+import java.util.Stack;
+import java.util.zip.GZIPInputStream;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+
+public class LambdaToS3 {
```
Comment (des-hippo) [inline tests/event-processing-tests/src/test/java/uk/gov/di/txma/eventprocessor/bdd/LambdaToS3.java:35]: Can we rename this to LambdaToS3StepDefinitions or something similar.

**Review by des-hippo — APPROVED** [review]
2 minor comments but LGTM.

**Review by des-hippo — APPROVED** [review]
Looks fab.

**Pull request merged**